src/App.tsx

```diff
@@ -1,42 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './components/Tabs/types/Tab';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+export const tabs: Tab[] = [
+  {
+    id:
+    'tab-1',
+    title: 'Tab 1',
+    content: 'First week with React. BTW the second too.',
+    cat: 1,
+  },
+  {
+    id: 'tab-2',
+    title: 'Tab 2',
+    content: 'Ihor said he was upset. OMG whats wrong with us, dudes.',
+    cat: 2,
+  },
+  {
+    id: 'tab-3',
+    title: 'Tab 3',
+    content: 'But we are sweet bulochki and can do it! Move your bulochki!',
+    cat: 3,
+  },
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+  const [darkMode, setDarkMode] = useState(false);
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  return (
+    <div className={cn(
+      'section',
+      {
+        'dark-mode': darkMode,
+        'light-mode': !darkMode,
+      },
+    )}
+    >
+      <div className="container">
+        <div className="switch-checkbox">
+          <label className="switch">
+            <input type="checkbox" onChange={() => setDarkMode(!darkMode)} />
+            <span className="slider round"> </span>
+          </label>
+        </div>
+      </div>
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+      <h1
+        className={cn(
+          'title',
+          {
+            'dark-mode--title': darkMode,
+          },
+        )}
+      >
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
+        selectedOnTab={setSelectedTab}
+      />
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <img
+        src=""
+        alt=""
+        className={`cat${selectedTab.cat}`}
+      />
```

Comment:
I think it is better to use div for background image. I am not sure, but is seems that your images are broken, because your tag <img> doesn't have src. + If you use <img>, you need to specify alt for better SEO.

Reply:
![image](https://user-images.githubusercontent.com/102512764/212267170-3be00938-0e45-4482-a433-f2bfd6c9f504.png)

Reply:
When I tried to add it, the picture breaks. Perhaps the reason is the small weight of the images 🤔